build_deps/toolchains/gcc7_manylinux2010-nvcc-cuda11/BUILD

```diff
@@ -57,9 +57,9 @@ cc_toolchain_config(
         "/dt7/usr/lib/gcc/x86_64-pc-linux-gnu/7/include",
         "/dt7/usr/lib/gcc/x86_64-pc-linux-gnu/7/include-fixed",
         "/dt7/usr/include",
-        "/usr/local/cuda-10.1/targets/x86_64-linux/include",
-        "/usr/local/cuda-10.1/include",
-        "/usr/local/cuda-10.1/extras/CUPTI/include",
+        "/usr/local/cuda-11.0/targets/x86_64-linux/include",
```

Comment:
Would it be possible when someone successfully builds TFRA with cuda version other than 11.0, or specify the cuda version according to the Tensorflow version.